packages/algorand-js/test/src/interpreter/opcode-list.ts

```diff
@@ -2346,14 +2348,12 @@ describe("Teal Opcodes", function () {
 
         const appId = interpreter.runtime.ctx.tx.apid;
         const acc = interpreter.runtime.ctx.state.accounts.get(elonAddr) as StoreAccountI;
-        let localStateCurr = acc.appsLocalState.get(appId)?.["key-value"];
+        let localStateCurr = acc.appsLocalState.get(appId)?.[keyValue];
         assert.isDefined(localStateCurr);
```

Comment:
can we move this part to appsLocalState function or as how TEAL behaves in this case ?

Reply:
This is just an extra assert, because map can also return undefined value.